### ISSUE TITLE
Provide Pull Request icons for Closed

### ISSUE THREAD

**Issue by nschonni** [primer/octicons-v2#301]
Similar to how "issues" has the closed icon with the checkmark, there should be a non-colour based way to differentiate open/closed pull requests.

This would improve the accessibility for those that have trouble differentiating colours https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html

/cc @muan for the accessibility opinion

**Comment by nschonni** [comment]
Thanks @ashygee! maybe if this is the better repo, https://github.com/primer/octicons/issues/376 should get transferred as well

**Comment by edokoa** [comment]
I was working on a few versions and I think that what will work in this case is having the checkmark in the bottom right side of the icon, as the top left will cover the arrow, which is fundamental to understand PR.

This also opens the door to using an "X" version in case we need a rejected or failed version.

![image](https://user-images.githubusercontent.com/694831/75991102-3e651880-5ef6-11ea-9fc3-5192290a1266.png)

**Comment by nschonni** [comment]
Maybe a pencil for the Draft icon in the other issue would work in the same spot.
My only concern with the check/cross is if people would confuse them with CI status of the PR

**Comment by edokoa** [comment]
Yes, we were also worried about this. What do you think @ashygee @colebemis ?

**Comment by auareyou** [comment]
Latest feedback regarding this issue: https://github.com/github/accessibility/issues/28

**Comment by nschonni** [comment]
@auareyou sorry it looks like that repo is private :(

**Comment by auareyou** [comment]
Ooops! Sorry. I'm linking the ticket with the feedback directly here then, is that ok? https://github.zendesk.com/agent/tickets/605458

**Comment by auareyou** [comment]
Yes! Sorry about that. We'll just keep it linked as a reference. Thank you!

**Comment by nschonni** [comment]
I wonder if you looked at a diagonal line through the PR icon, like you've done in the new "mute" icon
